### ISSUE TITLE
[Bug]: Date slider not draggable

### ISSUE THREAD

**Issue by teemutontti** [Pearl-image-comparer/Pearl#69]
Trying to drag the date slider which is on top of the Leaflet map doesn't work. It just snaps to the next value and this is not what we want.

### To reproduce
1. Once in the app open the satellite view by clicking the satellite toggle button.
2. Click on the slider handle and try to drag it.
3. As you can see it only snap in to the second value and won't let you drag it.

### Expected behavior

The slider should be smoothly dragged around.

### Preview

https://github.com/user-attachments/assets/adc6fc5c-0960-4552-b1f0-e8e1d84bc336

### Additional context

One previous problem was with Leaflet map being dragged when trying to drag the slider. This was fixed by disabling map drag with `map.dragging.disable()` once slider is interacted with. Could be related to this problem.

**Comment by kuukkanen** [comment]
Maybe adding a `<div />` or something behind the slider as to make the map not interactive underneath would resolve both issues? It can be made invisible but still receive mouse inputs. Also, don't know if related, but having a little debounce for the map updates would help, especially when dragging. That would reduce the API calls tremendously when the user is selecting a specific date because it's not going to fetch any unnecessary images for the dates user dragged over.

**Comment by teemutontti** [comment]
Adding a `<div />` didn't work 😞 I also tried `stopPropagation()` but that also had no effect. I found some kind of solution... Changing the `onChange`-method to `onChangeCommitted` affects the functionality so, that you can slide the slider freely and the change comes into affect only after you release the slider. The handle doesn't visually move at all (which is unfortunate) but it works fine. Here's an example: 

https://github.com/user-attachments/assets/11393cf4-cec3-4368-a2b8-8f5ae763d74f

Other way to do this is to create the slider using `L.DomUtil.create()`, which doesn't have any of the problems we have faced with custom React-components on top of the Leaflet map. The code would look something like this: 
```
const sliderControl = L.Control.extend({
  options: {
    position: "topright",
  },
  onAdd: function () {
    const container = L.DomUtil.create("div", "leaflet-control-slider");
    L.DomEvent.disableClickPropagation(container);

    const slider = L.DomUtil.create("input", "", container);
    slider.type = "range";
    slider.min = "1";
    slider.max = "100";
    slider.value = value.toString();
    slider.addEventListener("input", (e) => {
      console.log("Slider value: ", e.target.value);
    });

    return container;
  },
});

const controlInstance = new sliderControl();
map.addControl(controlInstance);
```

**Comment by teemutontti** [comment]
The slider can now be dragged using `onChangeCommitted`!🥳 

There still is a small issue where the user can't see the visual change in the slider thumb when it's being dragged, but the value is being set to the correct date when the slider is released. Maybe this visual problem could be addressed in a separate issue?
